### PR TITLE
Fix ResponseAfterNttpEnabledTestCase

### PR DESCRIPTION
**Pull request by sachithKay**

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  ~
+-->
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="response_nhttp_synapse_proxy" transports="https http" startOnLoad="true" trace="disable">
+        <description/>
+        <target>
+            <inSequence>
+                <log level="custom">
+                    <property name="at___insequence" value="===============its here"/>
```
Comment (Nirothipan) [inline integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml]: let's remove these, AFAIR these weren't there earlier.

===============its here :)

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                <log level="full"/>
+            <send>
+                <endpoint>
+                    <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
+                </endpoint>
+            </send>
+            </inSequence>
+            <outSequence>
+                   <log level="full"/>
+            <send/>
```
Comment (milindaperera) [inline integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml]: Formatting issue here. Let's reformat entire file

**Comment by Nirothipan** [comment]
Also lets rename this class( correct the spelling mistake ) as we discussed offline .

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml (hunk 2 above)
Comment (sachithKay) [reply integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml]: done

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml (hunk 1 above)
Comment (sachithKay) [reply integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/response_nhttp_synapse_proxy.xml]: done :)

**Pull request merged**